core-feature-pack/galleon-common/src/main/resources/packages/core.docs.licenses.merge/pm/wildfly/merger.xsl

```diff
@@ -59,36 +62,19 @@
     </xsl:template>
 
     <!-- FILE PROCESSING -->
-      
-    <xsl:template name="startDocument">
-        <xsl:text disable-output-escaping="no">
-</xsl:text>
-        <xsl:text disable-output-escaping="yes">&lt;licenseSummary&gt;</xsl:text>
-        <xsl:text disable-output-escaping="no">
-  </xsl:text>
-        <xsl:text disable-output-escaping="yes">&lt;dependencies&gt;</xsl:text>
-    </xsl:template>
-
-    <xsl:template name="endDocument">
-            <xsl:text disable-output-escaping="no">
-  </xsl:text>
-        <xsl:text disable-output-escaping="yes">&lt;/dependencies&gt;</xsl:text>
-        <xsl:text disable-output-escaping="no">
-</xsl:text>
-        <xsl:text disable-output-escaping="yes">&lt;/licenseSummary&gt;</xsl:text>
-    </xsl:template>
 
     <xsl:template name="processFile">
         <xsl:param name="fileName"/>
         <xsl:for-each select="document($fileName)/licenseSummary/dependencies/*">
-            <xsl:text disable-output-escaping="no">
-    </xsl:text>
             <xsl:copy>
-                <xsl:apply-templates select="@* | node()"/><xsl:text disable-output-escaping="no">  </xsl:text><source><xsl:call-template name="substring-after-last">
-                    <xsl:with-param name="string" select="$fileName"/>
-                    <xsl:with-param name="delimiter" select="$fileSeparator"/>
-                </xsl:call-template></source><xsl:text disable-output-escaping="no">
-    </xsl:text>
+                <xsl:apply-templates select="@* | node()"/>
+                <xsl:element name="source">
+                    <xsl:attribute name="from">core-feature-pack/galleon-common</xsl:attribute>
```

Comment:
The attribute from was not added to the WildFly Full variant, is it expected here? it looks like even less relevant since WildFly Core only uses a single feature pack.

I don't think an additional attribute could affect the license processes done before, but well, just wanted to know the reason for this difference.

Reply:
I've deleted it, I think that was a leftover from me testing how the licenses propagate to full